crates/sui-protocol-config/src/lib.rs

```diff
@@ -1327,6 +1327,9 @@ pub struct ProtocolConfig {
     /// Adds an absolute cap on the maximum transaction cost when using TotalGasBudgetWithCap at
     /// the given multiple of the per-commit budget.
     gas_budget_based_txn_cost_absolute_cap_commit_count: Option<u64>,
+
+    /// SIP-45: K in the formula `amplification_factor = max(0, gas_price / reference_gas_price - K)`.
+    sip_45_k: Option<u64>,
```

Comment:
Let's call this `consensus_amplification_threshold` or something more meaningful.